spec/lib/traffic_source_parser/parser/utmz_parser_spec.rb

```diff
@@ -25,7 +25,7 @@
           let(:campaign) { '(referral)' }
           let(:medium) { 'referral' }
           let(:channel) { 'Social' }
-          let(:source) { 't.co' }
+          let(:source) { 'Twitter' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.